### ISSUE TITLE
Warning after updating to latest build tools in 17.6

### ISSUE THREAD

**Issue by ErikEJ** [VsixCommunity/Community.VisualStudio.Toolkit#432]
Not sure if this is the right place, but after ugrade and update to build tools 17.6 I get this:

```
Severity	Code	Description	Project	File	Line	Suppression State	Priority
Warning		The extension is incompatible with the targeted version of Visual Studio. More info at https://aka.ms/ExtensionSdkErrors. 
Following references are incompatible.
Microsoft.VisualStudio.Web.BrowserLink.12.0.dll -> envdte	EFCorePowerTools	C:\Users\xxx\.nuget\packages\microsoft.vssdk.compatibilityanalyzer\17.6.2164\build\Microsoft.VsSDK.CompatibilityAnalyzer.targets	36		Normal
```

**Comment by RobertvanderHulst** [comment]
> I recommend staying away from the 17.6 build tools for now, as I have experienced other bugs (SDK files were included in the vsix file)

Confirmed. I have installed the 17.4 SDK folder inside 17.6 and then it works as expected.

**Comment by ErikEJ** [comment]
@madskristensen @AArnott Just tested the 17.7 build tools, same warning and same issue with SDK files included in the VSIX. Am I doing something wrong? (Hoping this does not affect everyone)

**Comment by RobertvanderHulst** [comment]
@ErikEJ This is fixed in 17.8. It is reported to be fixed in Fixed in Visual Studio 2022 version 17.8 Preview 2
See https://developercommunity.visualstudio.com/t/Visual-Studio-SDK-for-1761-includes-to/10373728?port=1025&fsid=db16ddd0-215d-4eef-aaf1-3c4b7ed1ee47&ref=native&refTime=1697456276415&refUserId=ff417d99-e25a-4e89-8765-29d305833f98

**Comment by ErikEJ** [comment]
Confirmed fixed in 17.8 build tools

**Comment by ErikEJ** [comment]
Aaand back in the 17.9 build tools 😢

**Comment by ErikEJ** [comment]
@AArnott @madskristensen FYI

**Comment by ErikEJ** [comment]
Finally fixed in 17.11 - phew
